executable/Env.re

```diff
@@ -25,8 +24,66 @@ let rec makeTemporarySymlink = () => {
   };
 };
 
-let run = (~shell, ~multishell, ~nodeDistMirror, ~fnmDir) => {
+let rec printUseOnCd = (~shell) =>
+  switch (shell) {
+  | System.Shell.Bash => {|
+    __fnmcd () {
+      cd $@
+
+      if [[ -f .node-version && .node-version ]]; then
+        echo "fnm: Found .node-version"
+        fnm use
+      elif [[ -f .nvmrc && .nvmrc ]]; then
+        echo "fnm: Found .nvmrc"
+        fnm use
+      fi
+    }
+
+    alias cd=__fnmcd
+  |}
+  | Fish => {|
```

Comment:
Hey @elliottsj! do you want to take a look? I know it works - but do you think this is a good way to do it on fish?

Reply:
Looks good to me! It's a little beyond my expertise, but from a cursory Google search, `--on-variable PWD` seems like a great way to do this.